packages/vitetris.rb

```diff
@@ -0,0 +1,27 @@
+require 'package'
+
+class Vitetris < Package
+  description 'Classic multiplayer tetris for the terminal'
+  homepage 'https://www.victornils.net/tetris/'
+  version '0.59.1'
+  license 'BSD-2-Clause'
+  compatibility 'all'
+  source_url 'https://github.com/vicgeralds/vitetris.git'
+  git_hashtag "master"
+
+  depends_on "curl"
+  depends_on "llvm"
+  depends_on "make"
+
+  def self.build
+    system "git checkout 9bbea4f5ab35d6fb8fe70c14daba19e7f35ab44e"
+    system 'make -s CFLAGS="-O2 -pipe -Wno-error=implicit-int -Wno-error=implicit-function-declaration"'
+  end
+
+  def self.install
+    system "make -s install"
+  end
+  def self.postinstall
+    puts "Vitetris successfully installed. You can run the game with \"tetris\""
```

Comment:
Take a look at our other postinstall messages to see how we format them.

Reply:
Use `ExitMessage.add` instead of `puts` to ensure the postinstall message is added to the end.